Formula/safe.rb

```diff
@@ -0,0 +1,40 @@
+class Safe < Formula
+  desc "Vault CLI"
+  homepage "https://github.com/starkandwayne/safe"
+  url "https://github.com/starkandwayne/safe/archive/v0.8.1.tar.gz"
```

Comment:
Looks like there's an update available here now.

Reply:
ok; i'll push an update later this afternoon.

Reply:
@DomT4 update pushed!  i believe i've addressed your latest concerns, including removing the `gox` dependency (looking at it again i agree with you)